doc/.templates/layout.html

```diff
@@ -1,34 +1,45 @@
 {% extends "!layout.html" %}
 
-{%- block rootrellink %}
-<li>
-    <span class="version_switcher_placeholder"></span>
-</li>
-{{ super() }}
-{% endblock %}
-
-{%- block extrahead %}
+{% block footer %}
 {{ super() }}
 <script type="text/javascript">
   var _gaq = _gaq || [];
   _gaq.push(['_setAccount', 'UA-168290-9']);
   _gaq.push(['_trackPageview']);
-</script>
 
-<script type="text/javascript"
-    src="{{ pathto('_static/version_switch.js', 1) }}">
+  (function() {
+    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
+    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
+    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
+  })();
 </script>
-{% endblock %}
 
-{% block footer %}
-{{ super() }}
+<!--Insert version switcher at bottom of sidebar-->
 <script type="text/javascript">
-  (function() {
-    var ga = document.createElement('script');
-    ga.src = ('https:' == document.location.protocol ?
-              'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
-    ga.setAttribute('async', 'true');
-    document.documentElement.firstChild.appendChild(ga);
-  })();
+  var span = document.createElement("SPAN")
+  var span_class = document.createAttribute("class");
+  span_class.value = "version_switcher_placeholder";
+  span.setAttributeNode(span_class);
+    
+  var div = document.createElement("DIV");
+  var text = document.createTextNode("v:");
```

Comment:
@slefrancois This, along with [L35](https://github.com/Theano/Theano/pull/4604/files#diff-6f2b65c406aa07382144f10909a074f9R35), seems to be the source of the extra `v:` instead.